README.md

```diff
@@ -21,16 +22,15 @@ In order to build and run this service you need the node package manager install
 
 Install purescript, bower and pulp:
 ```bash
-npm install -g purescript bower pulp
+yarn global add purescript bower pulp
 ```
-_Note:_ If you get an error message saying one (or more) of the packages failed to install, check you nodejs and npm versions.  For example, the default repos for Ubuntu (on 2017-07-21) have a bit older nodejs and npm versions, and the install failed.  Upgrading to node.js v6.11.1 and npm v3.10.10 and running with Administrator privileges allowed the install to succeed.  One possible upgrade path is to use a PPA.
```

Comment:
We probably still need this message, but I guess you can drop the references to NPM and keep the NodeJS mentions. And we'll want to check the version numbers (maybe just mention what you machine has).